### ISSUE TITLE
Breakage when minifying at-import with unquoted urls containing semicolons

### ISSUE THREAD

**Issue by zytzagoo** [tubalmartin/YUI-CSS-compressor-PHP-port#45]
Here's a testcase that fails with current master:

```css
@import url(//example.org/foo.css?a&#038;b);
@import url(//example.org/bar.css);
```

Expected output:
```css
@import url(//example.org/foo.css?a&#038;b);@import url(//example.org/bar.css);
```

Actual output:
```css
@import url(//example.org/foo.css?a&#038;@import url(//example.org/bar.css);b);
```

Diff highlight:
```diff
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'@import url(//example.org/foo.css?a&#038;b);@import url(//example.org/bar.css);'
+'@import url(//example.org/foo.css?a&#038;@import url(//example.org/bar.css);b);'
```

When urls are quoted, the problem doesn't occur (because `processStringsCallback()` captures and later restores the url "tokens").
However, I cannot control the inputs being given to `Minifier`.

Do you have any pointers on how to best approach fixing this?
Is this something you think needs fixing at all?

I have a branch [ready for a PR over at my fork (with the test suite passing)](https://github.com/tubalmartin/YUI-CSS-compressor-PHP-port/compare/master...zytzagoo:at-import-unquoted-urls?expand=1), as this is currently a showstopper for getting the latest version integrated into Autoptimize, however, there's probably a better/smarter way to go about fixing this -- I just needed to get something that works quickly...

**Comment by tubalmartin** [comment]
Thank you for reporting the issue extensively. I will review this for sure :)

**Comment by tubalmartin** [comment]
Umm yes this is a valid issue. A semicolon is a reserved character in an URI and CSSmin should support it in import at rules. I'll fix this.

**Comment by tubalmartin** [comment]
Fixed in v4.1.1
